docs/content/configuration/creating_pvc.md

```diff
@@ -18,12 +18,12 @@ The `persistentvolumeclaim/<pvc-name> created` message is emitted.
 
 Use the following sections, according to your PVC needs:
 
-- Creating PVC for volume with file system
-- Creating PVC for raw block volume
+- [Creating PVC for volume with `Filesystem`](#creating-pvc-for-volume-with-filesystem)
+- [Creating PVC for raw block volume](#Creating-PVC-for-raw-block-volume)
 - Creating PVC from volume snapshot
 - Creating a volume clone from an existing PVC
 
-## <a name=PVC-fs></a>Creating PVC for volume with Filesystem
+## Creating PVC for volume with `Filesystem`
```

Comment:
the other title is "Creating PVC for raw block volume", and not "creating PVC for volume with `Block`" (which I think sounds awkward).
should this be aligned to something like `Creating PVC for file system volume`/`Creating PVC for a volume with a file system`?
or maybe we could change them both to e.g. "Creating PVC for `Block` volume" & "Creating PVC for `Filesystem` volume"

Reply:
continuing [there](https://github.com/IBM/ibm-block-csi-driver/pull/415/files#r767239504)